content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -21,33 +21,60 @@ further_reading:
 
 - The .NET Tracer library for Datadog is open source. For more information, see the [.NET Tracer repository][1].
 
+## Runtime support policy for .NET Core APM
+
+Datadog APM for .NET Core is built upon dependencies defined in specific versions of the host operating system, .NET Core runtime, certain .NET Core libraries, and the Datadog Agent/API. When these versions are no longer supported by their maintainers, Datadog APM for .NET Core limits its support for these as well.
+
+### Levels of support
+
+delner marked this conversation as resolved.
```

Comment:
```suggestion
```

Reply:
Fixed in 357cc23292e321b0326fa74873bfb42752d2e27d